apps/generic_terminal/unix_shell.talon

```diff
@@ -0,0 +1,26 @@
+tag: terminal
+tag: user.generic_unix_shell
+os: mac
+os: linux
+-
+# Generic Shell Commands. (General convension is to use the command name in "man" when possible)
+
+^change (directory|dir)$: "cd "
+^home (directory|dir)$: "~/"
+^copy$: "cp "
+^make dir$: "mkdir "
+^cancel$: key(ctrl-c)
+^(move|rename) command: "mv "
+^list directory$: "ls "
+^list current directory$: user.terminal_list_directories()
```

Comment:
We already have the `lisa` command in `generic_terminal.talon`.

Reply:
Yes I saw it. 
But Katie and Lisa don't mean anything. 
If I added "netstat" or "which" what would be the convention to name those commands? "David" or "Michael"?
How would one remember those commands? 
I can sort of see some sound similarity between ls and lisa, but what is the similarity or relationship between katie and "change directory." 
I am in favor of replacing lisa and katie in favor of more standard and scalable command system. 
And if someone wants to remap them locally to names that mean something to them specifically they should feel free to do so.

Reply:
fwiw Katie came out of "see dee" not being reliable, so became "Kay dee". convenient attr time they were both names *shrug*

but I agree, it's not particularly sensible to most people. something generic is better

I've since changed to more explicit commands in my repo, with prefixes that roughly match what I'm working on, like file, folder, disk, net:

https://github.com/fidgetingbits/knausj_talon/blob/master/apps/platforms/linux/commandline.talon

I find it works well. things like:

file list
file touch 
file edit
folder create

might be worth considering